<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartSettingDialog</class>
 <widget class="QDialog" name="ChartSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chart Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="MainVerticalLayout">
     <item>
      <widget class="QGroupBox" name="LimitGroupBox">
       <property name="title">
        <string>Default Limits Setting</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="LimitsHorizontalLayout">
          <item>
           <widget class="QLabel" name="HighLimitLabel">
            <property name="text">
             <string>High Limit</string>
            </property>
            <property name="buddy">
             <cstring>HighLimitLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="HighLimitLineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="LowLimitLabel">
            <property name="text">
             <string>Low Limit</string>
            </property>
            <property name="buddy">
             <cstring>LowLimitLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LowLimitLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GroupCountGroupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="HistoCountVerticalLayout">
          <item>
           <layout class="QHBoxLayout" name="MaxCountHorizontalLayout">
            <item>
             <widget class="QLabel" name="MaxHistogramGroupLable">
              <property name="text">
               <string>Max Histogram Group Count</string>
              </property>
              <property name="buddy">
               <cstring>MaxCountSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="MaxCountSpinBox">
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="value">
               <number>99</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="MinCountHorizontalLayout">
            <item>
             <widget class="QLabel" name="MinHistogramGroupLable">
              <property name="text">
               <string>Min Histogram Group Count</string>
              </property>
              <property name="buddy">
               <cstring>MinCountSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="MinCountSpinBox">
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="value">
               <number>13</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChartSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChartSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
